pkg/deploys/fuse/objects.go

```diff
@@ -62,6 +63,9 @@ func getNamespaceObj(id string) *corev1.Namespace {
 	return &corev1.Namespace{
 		ObjectMeta: metav1.ObjectMeta{
 			Name: id,
+			Labels: map[string]string{
+				"monitoring-key": os.Getenv("MONITORING_KEY"),
```

Comment:
I think it would be better to pull this out in the main.go, and pass it through to where it's needed. Pulling it directly from env like this makes it more difficult to write tests for.

Reply:
Agreed. I'll update this. Thanks.